CONTRIBUTING.MD

```diff
@@ -58,10 +58,15 @@ Every change must be described with a change log: Run "rush change" on your comm
 ### Code Change Workflow
 
 1. Make source code changes on a new Git branch.
-2. Ensure unit tests pass when run locally: `rush test`.
-3. Commit your changes.
-4. Add a changelog entry (which could potentially cover several commits): `rush change`.
-5. Follow prompts to enter a change description or press ENTER if the change does not warrant a changelog entry. If multiple packages have changed, multiple sets of prompts will be presented.
-6. Completing the `rush change` prompts will cause new changelog entry JSON files to be created.
-7. Commit the changelog JSON files.
-8. Publish changes to the branch and open a pull request.
+1. Ensure unit tests pass when run locally: `rush test`.
```

Comment:
all the steps show up as step 1.

Reply:
This will still be rendered correctly in the HTML. [Google Markdown style guide best practices](https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists) says long lists should use the lazy numbering method. I figure since this is up to 10 and has a sub-list the lazy method would be better.

Reply:
But, I can change it back since it makes it a bit more difficult to read in source code.

Reply:
i was unaware of this, good to know!